### ISSUE TITLE
The problem Of muscle hypertrophy: Revisited

### ISSUE THREAD

**Issue by keiya-k** [keiya-k/Awesome-Scholars#5]
There was an interesting research that pointed out the problem with machos ([1](https://pubmed.ncbi.nlm.nih.gov/27717161/))

This is a review paper that summarized past research on "muscle hypertrophy and muscle strength". Basically, is it true that the thicker the arm, the more it can lift?

So, the main points of the research were:

- Muscle thickness and strength have a really small correlation. Increasing muscle size and increasing strength are different phenomena. 

- If you skip muscle training, the thickness of the muscle tends to shrink, but the muscle strength tends to be maintained. It suggests that even 1 high-load training a month is enough to maintain your muscle strength.

- There are many cases where muscle strength increases even though muscle size is not. 

- The amount of weight used and the thickness of the muscle are not correlated

- If the load of muscle training increases, muscle strength will definitely increase

That's about it. No matter how big the muscles are, it doesn't mean they are strong. I guess that's why i sometimes see relatively slim people bench pressing 100kg.

Difference Between Increasing Muscle Mass and Increasing Muscle Strength

- If you want to make your muscles thicker = tire your muscles and stimulate them with any light load

- If you want to build strength = train with a high load and aim to increase your 1RM (maximum weight you can lift just once)
 

How to Train for Muscle Mass or Strength

According to the researcher:

"Consider low-load muscle training (around 20 to 30% of maximum muscle strength) and high-load muscle training (70% or more of maximum muscle strength).

Regardless of which method you use, if you do 2-3 sets and stimulate your muscles, the size of the muscles will increase. However, the development of muscle strength will be different.

For example, let's say someone has been doing only low-load muscle training. If you give that person a strength test, they will definitely lose to someone who has been training with high-load."

In other words:

- If you want to make your body bigger: focus on light-load training and increase your overall volume of training (volume = sets x reps x weight). Arnold Schwarzenegger also said that he mainly did light-load training.

- If you want to become a functional slim macho: focus on high-load training. Maybe it would be best to use a technique such as HIT (a high-intensity short-time training).

That's about it. In my case, i mainly do high-load training but switch to low-load a few times a month.
